control-plane/dist/add/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
```

Comment:
Please remove this since we don't support adding jobs in the control plane.

Reply:
Done